packages/dashboard/src/components/tasks/tasks-app.tsx

```diff
@@ -309,15 +309,19 @@ export const TasksApp = React.memo(
       const exportTasksToCsv = async (minimal: boolean) => {
         AppEvents.loadingBackdrop.next(true);
         const now = new Date();
-        const allTasks = await getAllTasks(now);
-        const allTaskRequests = await getAllTaskRequests(allTasks);
-        if (!allTasks || !allTasks.length) {
+        const pastMonthTasks = await getPastMonthTasks(now);
+
+        // FIXME: Task requests are currently required for parsing pickup and
+        // destination information. Once we start using TaskState.Booking.Labels
+        // to encode these information, we can skip querying for task requests.
+        const pastMonthTaskRequests = await getPastMonthTaskRequests(pastMonthTasks);
+        if (!pastMonthTasks || !pastMonthTasks.length) {
           return;
         }
         if (minimal) {
-          downloadCsvMinimal(now, allTasks, allTaskRequests);
+          downloadCsvMinimal(now, pastMonthTasks, pastMonthTaskRequests);
```

Comment:
1. "download*" is misleading since it doesn't actually download, suggest to rename it to "save*" or add docs.
2. It's weird that "minimal" contains info that is not in "full" (from what I see, pickup and dropoff), I will say to include the task request in `full` so it has the same data.
3. Parsing the task request for pick and dropoff is also a red flag, it is out of scope of this PR so just add a `FIXME`. In future PR we can look at using task tags to store this information and just dump it, it also solves the issue of requiring double query to get all the required data.

Reply:
1. Changing it to `export`, https://github.com/open-rmf/rmf-web/pull/909/commits/651562d7f1db787161725b79a9198f5cec0cd4e5

2. Technically, `full` contains all the information of `minimal` and much more, since any number of  pickups and dropoffs can be parsed from the task phases if needed. And since we will be adding pick and dropoff information in labels soon, I will not make any changes to this. 

3. FIXME was added previously https://github.com/open-rmf/rmf-web/pull/909/commits/d1267670d119315eecfd86fed18ecb5b81b449c4

Reply:
> Technically, full contains all the information of minimal and much more, since any number of pickups and dropoffs can be parsed from the task phases if needed. And since we will be adding pick and dropoff information in labels soon, I will not make any changes to this.

Then we don't need to fetch task requests for "minimal"? Or do you mean that it will be included after we need the tags? If so then add a `TODO`.